### DISCUSSION TITLE
v0.11.0

### DISCUSSION THREAD

**Discussion by WithoutPants** [stashapp/stash#2012] [Announcements]
# Release Notes
### 💫 [Help Shape the Future of Stash!](https://forms.gle/x5nZa1zrVTJpgMHx8)
The Stash developers would greatly appreciate if you take a [short, anonymous survey](https://forms.gle/x5nZa1zrVTJpgMHx8). It would help us out a great deal to make yourself heard, let us know how you use Stash, and tell us what you'd like to see in the future.

### ✨ New Features
* Added Identify task to automatically identify scenes from stash-box/scraper sources. See manual entry for details. ([#1839](https://github.com/stashapp/stash/pull/1839))
* Added support for matching scenes using perceptual hashes when querying stash-box. ([#1858](https://github.com/stashapp/stash/pull/1858))
* Generalised Tagger view to support tagging using supported scene scrapers. ([#1812](https://github.com/stashapp/stash/pull/1812))
* Added built-in `Auto Tag` scene scraper to match performers, studio and tags from filename - using AutoTag logic. ([#1817](https://github.com/stashapp/stash/pull/1817))
* Add options to auto-start videos when playing from selection and continue to scene playlists. ([#1921](https://github.com/stashapp/stash/pull/1921))
* Support is (not) null for multi-relational filter criteria. ([#1785](https://github.com/stashapp/stash/pull/1785))
* Optionally open browser on startup (enabled by default for new systems). ([#1832](https://github.com/stashapp/stash/pull/1832))
* Support setting defaults for Delete File and Delete Generated Files in the Interface Settings. ([#1852](https://github.com/stashapp/stash/pull/1852))
* Added interface options to disable creating performers/studios/tags from dropdown selectors. ([#1814](https://github.com/stashapp/stash/pull/1814))

### 🎨 Improvements
* Added Italian 🇮🇹, French 🇫🇷, and Spanish 🇪🇸 translations ([#1875](https://github.com/stashapp/stash/pull/1875), [#1967](https://github.com/stashapp/stash/pull/1967), [#1886](https://github.com/stashapp/stash/pull/1886))
* Added stash-id to scene scrape dialog. ([#1955](https://github.com/stashapp/stash/pull/1955))
* Reworked main navbar and positioned at bottom for mobile devices. ([#1769](https://github.com/stashapp/stash/pull/1769))
* Show files being deleted in the Delete dialogs. ([#1852](https://github.com/stashapp/stash/pull/1852))
* Added specific page titles. ([#1831](https://github.com/stashapp/stash/pull/1831))
* Show pagination at top and bottom of page. ([#1776](https://github.com/stashapp/stash/pull/1776))
* Include total duration/megapixels and filesize information on Scenes and Images pages. ([#1776](https://github.com/stashapp/stash/pull/1776))
* Optimised generate process. ([#1871](https://github.com/stashapp/stash/pull/1871))
* Added clear button to query text field. ([#1845](https://github.com/stashapp/stash/pull/1845))
* Moved Performer rating stars from details/edit tabs to heading section of performer page. ([#1844](https://github.com/stashapp/stash/pull/1844))
* Optimised scanning process. ([#1816](https://github.com/stashapp/stash/pull/1816))

### 🐛 Bug fixes
* Fix tag hierarchy not being validated during tag creation. ([#1926](https://github.com/stashapp/stash/pull/1926))
* Fix tag hierarchy validation incorrectly failing for some hierarchies. ([#1926](https://github.com/stashapp/stash/pull/1926))
* Fix exclusion pattern fields losing focus on keypress. ([#1952](https://github.com/stashapp/stash/pull/1952))
* Include stash ids in import/export. ([#1916](https://github.com/stashapp/stash/pull/1916))
* Fix tiny menu items in scrape menu when a stash-box instance has no name. ([#1889](https://github.com/stashapp/stash/pull/1889))
* Fix creating missing entities removing the incorrect entry from the missing list in the scrape dialog. ([#1890](https://github.com/stashapp/stash/pull/1890))
* Allow creating missing Studio during movie scrape. ([#1899](https://github.com/stashapp/stash/pull/1899))
* Fix image files in folder galleries not being deleting when delete file option is checked. ([#1872](https://github.com/stashapp/stash/pull/1872))
* Fix marker generation task reading video files unnecessarily. ([#1871](https://github.com/stashapp/stash/pull/1871))
* Fix accessing Stash via IPv6 link local address causing security tripwire to be activated. ([#1841](https://github.com/stashapp/stash/pull/1841))
* Fix Twitter value defaulting to freeones in built-in Freeones scraper. ([#1853](https://github.com/stashapp/stash/pull/1853))
* Fix colour codes not outputting correctly when logging to file on Windows. ([#1846](https://github.com/stashapp/stash/pull/1846))
* Sort directory listings using case sensitive collation. ([#1823](https://github.com/stashapp/stash/pull/1823))
* Fix auto-tag logic for names which have single-letter words. ([#1817](https://github.com/stashapp/stash/pull/1817))
* Fix huge memory usage spike during clean task. ([#1805](https://github.com/stashapp/stash/pull/1805))

<hr /><em>This discussion was created from the release <a href='https://github.com/stashapp/stash/releases/tag/v0.11.0'>v0.11.0</a>.</em>

**Comment by philpw99** [comment]
Just took the survey ! It was nicely done.
Lots of things we can look forward to. :smiley:
